src/what-if/scope_managementgroup.ts

```diff
@@ -0,0 +1,62 @@
+import { exec } from '@actions/exec';
+import { ExecOptions } from '@actions/exec/lib/interfaces';
+import { ParseOutputs, Outputs } from '../utils/utils';
+import * as core from '@actions/core';
+
+export async function WhatIfManagementGroupScope(azPath: string, region: string, template: string, deploymentName: string, parameters: string, managementGroupId: string, failOnStdErr: Boolean, excludeChangeTypes: String): Promise<Outputs> {
```

Comment:
this is almost a duplicate of DeployResourceGroupScope. Can't we just modify DeployResourceGroupScope and add `what-if` to deployment command?